utils/params.go

```diff
@@ -129,6 +129,13 @@ func (p *Project) setDefaultsIfNeeded() error {
 	return nil
 }
 
+func (p *Project) GetTechFromInstallCmdIfExists() []string {
+	if p.InstallCommandName != "" {
```

Comment:
I would add a check trying to convert into 'techutils.Technology' to make sure, adding a log if an issue occurred and skip this value